README.md

```diff
@@ -17,6 +17,7 @@ pvmigrate --source-sc default --dest-sc mynewsc
 |--------------------------|--------|----------|------------------|--------------------------------------------------------------------------------------------------|
 | --source-sc              | String | ✓        |                  | storage provider name to migrate from                                                            |
 | --dest-sc                | String | ✓        |                  | storage provider name to migrate to                                                              |
+| --namespace              | String |          |                  | namespace to find all PVCs                                                                       |
```

Comment:
```suggestion
| --namespace              | String |          |                  | namespace to find PVCs in                                                                       |
```
Can we change this to match the flag's description in https://github.com/replicatedhq/pvmigrate/pull/106/files#diff-4dea1cd5df620639401940fa831fde0d4364b551a2d69e30d1ca25c676e1378bR56 ?